views/trial-record.html

```diff
@@ -1,22 +0,0 @@
-{% extends "layouts/one-column.html" %}
```

Comment:
It's okay to remove it, as we don't link to it from anywhere yet. Just keep in mind we have this in git's history in case we'll need it in the future.